### DISCUSSION TITLE
OME Plugin Development

### DISCUSSION THREAD

**Discussion by adolfusadams** [OvenMediaLabs/OvenMediaEngine#1588] [Help]
Is it possible to write plugins that can work on Oven Media Engine?

**Comment by getroot** [comment]
OvenMediaEngine is open source, so any functionality can be added more reliably than with plugins.

**Reply by adolfusadams** [reply]
That is what I would like to know. Please indicate the way or ways to go about adding functionality to OvenMediaEngine.

**Reply by bchah** [reply]
What getroot is saying is that since OME is open-source you can submit any code changes you wish in a PR, even implementing a plugin system if you wish 😀
